db/migrate/20230315090426_add_scopes_to_token.rb

```diff
@@ -0,0 +1,14 @@
+class AddScopesToToken < ActiveRecord::Migration[7.0]
+  def change
+    add_column :tokens, :scopes, :jsonb, default: [], null: false
+    add_index  :tokens, :scopes, using: :gin
+
+    Token.all.each do |token|
```

Comment:
```suggestion
    Token.includes(:scopes).find_each do |token|
```

Le includes évite de faire du N+1 query, find_each fait du batch pour speed-up

Reply:
ok